wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/provisioncredential/EvidenceRequest.kt

```diff
@@ -660,8 +839,88 @@ fun EvidenceRequestIcaoNfcTunnelView(
             // data scanned from passport MRZ strip.
             IcaoMrtdCommunicationModel.Route.CAMERA_SCAN
         }
-    EvidenceRequestIcaoView(tunnelScanner, permissionTracker, initialRoute) {
-        provisioningViewModel.finishTunnel()
+
+    // if developer mode not enabled, simply show the MRZ or NFC screen without briefly showing
```

Comment:
nit: punctuation for these comments: start with a capital letter, end with a period.